composer.json

```diff
@@ -68,7 +68,7 @@
 		"wpackagist-plugin/amp": "^1.1.4",
 		"wpackagist-plugin/hummingbird-performance": "2.0.1",
 		"wpackagist-plugin/jetpack": "9.3.2",
-		"wpackagist-plugin/pdf-embedder": "^4.6",
```

Comment:
This is not needed in this PR

Reply:
Ok, I figured out that you updated this version to fix workflow errors, correct?

Reply:
Yes, it was updated because of CI error